src/utils/removeDateSuffix.m

```diff
@@ -2,27 +2,21 @@
 
 function removeDateSuffix(filenames, subjectPath)
     %
-    % Short description of what the function does goes here.
+    % Function removes the date suffix in a given filename(s)
+    % Function will look for regular expression in filenames with ``_date-``
+
     %
     % USAGE::
     %
-    %   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
-    %
-    % :param argin1: (dimension) obligatory argument. Lorem ipsum dolor sit amet,
-    %                consectetur adipiscing elit. Ut congue nec est ac lacinia.
-    % :type argin1: type
-    % :param argin2: optional argument and its default value. And some of the
-    %               options can be shown in litteral like ``this`` or ``that``.
-    % :type argin2: string
-    % :param argin3: (dimension) optional argument
-    % :type argin3: integer
-    %
-    % :returns: - :argout1: (type) (dimension)
-    %           - :argout2: (type) (dimension)
+    %   removeDateSuffix(filenames, subjectPath)
     %
-    % removeDateSuffix(filenames, subjectPath)
+    % :param filenames: File names with _date- suffix
+    % :type filenames: string
     %
+    % :param subjectPath: Location/path of the files
+    % :type subjectPath: string
     %
+    % :output: - files are renamed by removing '_date-yyyymmddHHMM' suffix
```

Comment:
```suggestion
    % :output: - files are renamed by removing '_date-*' suffix
```

Actually I was just checking the `cfg` structure and it is possible to change the format of that date to be something else than the default one you have here. So maybe let's put a "joker" in there.